docs/faq.md

```diff
@@ -109,6 +109,46 @@ commands =
 `tox` will not do any install. Poetry installs all the dependencies and the current package an editable mode.
 Thus, tests are running against the local files and not the built and installed package.
 
+#### Usecase #4
+```ini
+[tox]
+isolated_build = true
+
+[testenv]
+allowlist_externals = poetry
+commands_pre =
+    poetry install --no-root --sync --with test
+commands =
+    pytest tests/
```

Comment:
Any config made in the `[testenv]` is inherited to all other tox environment. Unless overwritten there, it will be executed there in the same way. For users that are not aware of this, it can lead to surprising behavior.

This is why I would prefer to have an own environment definition for tests as well, e.g. `[testenv:pytest]`.

Reply:
Yeah, I can get behind that, and I've seen plenty of projects write their configs that way for that reason. I'll make the update!

Reply:
I did this with `testenv:test` because I was trying to match the environment names to the dependency groups (on purpose), and to avoid putting tool names in there.

I'm happy to tweak it to `testenv:pytest` if you strongly prefer though, no strong opinion from me.